File: kinotic-orchestrator/src/main/java/org/kinotic/orchestrator/api/model/workload/NetworkMode.java
package org.kinotic.orchestrator.api.model.workload;

/**
 * Whether a {@link Workload}'s VM has network access.
 */
public enum NetworkMode {

    ENABLED,

    /**
     * No network access. The boxlite provider cannot boot a VM in this mode and rejects a
     * workload that asks for it; restrict egress with {@link NetworkPolicy#getAllowedHosts()}
     * instead.
     */
    DISABLED
}

File: kinotic-js/workspace/packages/os-api/src/api/model/workload/NetworkMode.ts

export enum NetworkMode {

    ENABLED = 'ENABLED',

    /**
     * No network access. The boxlite provider cannot boot a VM in this mode and rejects a
     * workload that asks for it; restrict egress with NetworkPolicy.allowedHosts instead.
     */
    DISABLED = 'DISABLED'
}
